sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ClientConstants.java

```diff
@@ -17,17 +17,25 @@ public final class ClientConstants {
     public static final String LINK_NAME_KEY = "linkName";
     public static final String ENTITY_PATH_KEY = "entityPath";
     public static final String ENTITY_NAME_KEY = "entityName";
+    public static final String UPDATED_LINK_CREDIT_KEY = "updatedLinkCredit";
+    public static final String REMOTE_CREDIT_KEY = "remoteCredit";
+    public static final String IS_PARTIAL_DELIVERY_KEY = "delivery.isPartial";
+    public static final String IS_SETTLED_DELIVERY_KEY = "delivery.isSettled";
     public static final String SESSION_NAME_KEY = "sessionName";
     public static final String FULLY_QUALIFIED_NAMESPACE_KEY = "namespace";
     public static final String OPERATION_NAME_KEY = "amqpOperation";
+    public static final String DELIVERY_KEY = "delivery";
     public static final String DELIVERY_STATE_KEY = "deliveryState";
+    public static final String DELIVERY_TAG_KEY = "lockToken";
     public static final String ERROR_CONDITION_KEY = "errorCondition";
     public static final String ERROR_DESCRIPTION_KEY = "errorDescription";
     public static final String EMIT_RESULT_KEY = "emitResult";
     public static final String SIGNAL_TYPE_KEY = "signalType";
     public static final String HOSTNAME_KEY = "hostName";
     public static final String INTERVAL_KEY = "interval_ms";
```

Comment:
I feel like this logic was here before, but we should try to be consistent with either using camel case or dashes or underscores to distinguish words.

Reply:
I agree, will change this to camel case for consistency.